### DISCUSSION TITLE
Doubts related to rules

### DISCUSSION THREAD

**Discussion by Aanal2901** [larics/icuas22_competition#26] [Q&A]
@fpetric
1) Are we supposed to complete both levels of difficulty or either one?
2) What is the time limit? Exceeding the time limit has been stated as one of the disqualifying rules, however time limit could not be found in the rulebook.

**Comment by fpetric** [comment]
hi @Aanal2901, 

thanks for asking! 

1. For the exploration difficulty, you can choose the difficulty level. If you ask for a map of the environment, you are automatically placed on the lower difficulty option, and if you do the task without the apriori map provided by us, you will be placed in the higher difficulty category. We are still working on the details, but we will probably make a service that you can call to request the map, and if the service is called will determine the difficulty level of the task.

2. For the time limit in the simulation part, we haven't defined it yet because we do not want to set it before we see how teams perform (we do not want to disqualify 80% of the teams, for example), so we will probably go with something like this:

- set a time limit after a few evaluation runs (we plan to evaluate first versions of your solutions starting from April 1st). This cutoff time will be defined as a percentage of the best time (similar to how Formula 1 does it with the cutoff time for the qualifying being 100% + X% of the fastest time). An example, if the fastest time observed was 100 seconds, and we opt for X=10, than the cutoff time would be 110 seconds. 

For the finals, the time limit will probably be defined by the battery :)
